includes/class-starter-content.php

```diff
@@ -327,14 +350,15 @@ private static function create_block( $block_type = null, $attributes = null, $c
 	 *
 	 * @return string Block markup.
 	 */
-	private static function add_featured_image( $post_id ) {
+	private static function add_featured_image( $post_id, $post_index ) {
```

Comment:
lintpick: [missing documentation for param](https://app.circleci.com/pipelines/github/Automattic/newspack-plugin/219/workflows/7a40e094-6403-4bf7-b6da-f518f4dbf52f/jobs/792/parallel-runs/0/steps/0-102)